exchanges/kraken/kraken_test.go

```diff
@@ -181,7 +181,7 @@ func TestFuturesSendOrder(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, k, canManipulateRealOrders)
 
-	_, err := k.FuturesSendOrder(context.Background(), order.Limit, futuresTestPair, "buy", "", "", "", order.ImmediateOrCancel, 1, 1, 0.9)
+	_, err := k.FuturesSendOrder(t.Context(), order.Limit, futuresTestPair, "buy", "", "", "", order.ImmediateOrCancel, 1, 1, 0.9)
```

Comment:
[test failures in this file](https://github.com/thrasher-corp/gocryptotrader/actions/runs/14859702956/job/41721262069?pr=1382#step:11:406)

Reply:
<img width="694" alt="Screenshot 2025-05-07 at 4 27 55 in the afternoon" src="https://github.com/user-attachments/assets/cfde8d25-7ad6-41eb-9564-e2a1041fb0ee" />
Added updated it to skip when encoutered with order.UnknownTIF value since it not not required